src/jit/emitarm.cpp

```diff
@@ -1436,6 +1436,18 @@ emitter::insSize emitter::emitInsSize(insFormat insFmt)
 
 /*****************************************************************************
  *
+ *  emitIns_valid_imm_for_vldst_offset() returns true when the immediate 'imm'
+ *   can be encoded as the offset in a vldr/vstr instruction.
+ */
```

Comment:
You could also add 
`// Returns true when 'imm' is a non-negative multiple of 4 that is less than 1024`